main.go

```diff
@@ -44,25 +45,36 @@ func init() {
 
 	utilruntime.Must(lokiv1beta1.AddToScheme(scheme))
 	// +kubebuilder:scaffold:scheme
-
-	// TODO: make this loading optional via a loki-operator CLI flag.
-	utilruntime.Must(monitoringv1.AddToScheme(scheme))
 }
 
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var enableServiceMonitors bool
+	var useTLSServiceMonitors bool
```

Comment:
:bulb: Let's be consistent here and use `enable` as a prefix for enabling feature flags.